src/lib/bcmp/bm/bristlemouth.h

```diff
@@ -1,21 +1,8 @@
 #pragma once
 
+#include "serial.h"
```

Comment:
why is serial.h needed here?

Reply:
Oops, I really have no idea how this got added :(